build.gradle

```diff
@@ -42,7 +45,8 @@ task validateDbConfig {
     description 'Ensure that the target database properties file exists'
     group 'Verification'
 
-    ext.dbConfig = file('src/main/resources/POSTGRES.properties')
+    def fileName = System.getProperty("geexport.db.info", "db-info.properties")
+    ext.dbConfig = file("src/main/resources/${fileName}")
```

Comment:
and we should refactor the code a bit (maybe not in this PR) to extract out constants and check for those configs.